webapp/src/modules/bid/reducer.spec.ts

```diff
@@ -0,0 +1,12 @@
+import { clearBidError } from "./actions"
+import { bidReducer, INITIAL_STATE } from "./reducer"
+
+describe("when clear bid error action is received", () => {
+  it("should set error field as null", () => {
+    const stateWithError = {
+      ...INITIAL_STATE,
+      error: 'Some test error'
+    }
+    expect(bidReducer(stateWithError, clearBidError())).toEqual(expect.objectContaining({ error: null }))
+  })
```

Comment:
Would you mind either defining the context in a before each or in the reducer call?
```suggestion
  it("should set error field as null", () => {
    expect(bidReducer({ ...INITIAL_STATE, error: 'Some test error' }, clearBidError())).toEqual(expect.objectContaining({ error: null }))
  })
```